### PR TITLE
- Added webadmin UI bookmark blog post.

### PR DESCRIPTION
**Pull request by awels**
Changes proposed in this pull request:

- Added a blog post about the new webadmin UI bookmarking capability.

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @mention yourself to sign) @awels

This pull request needs review by: @mykaul

### CODE REVIEW

**Comment by awels** [comment]
cc @vojtechszocs

File: source/blog/2018-01-18-ovirt-admin-bookmarks.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Direct linking to entities
+
+As a complementary functionality you can pass arguments to places that will execute some functionality based on the type of argument you have passed in. The following types are available:
+
+* SEARCH, is for main views only, this allows you to pre populate the search string used in the search bar.
+* NAME, most entities are uniquely named and you can use their name in a detail view to go directly to that named entity.
+* DATACENTER, quota and networks are not uniquely named, but are unique combined with their associated data center, to link directly to either you need to specify NAME and DATACENTER.
+* NETWORK, VNIC profiles are not uniquely named, but need both DATACENTER and NETWORK to be specified to directly link to it. 
+
+This functionality allows the bookmarks to work and take you directly to the entity/search in your bookmark. This also allows external applications to direct link to entities in the webadmin.
```
Comment (vojtechszocs) [inline source/blog/2018-01-18-ovirt-admin-bookmarks.md]: Please mention that this also works even if the user isn't already logged in, I'd replace the last sentence with following:

"
If the user isn't already logged in, they will be redirected to oVirt SSO login page and then back to the desired place in the application. This allows external applications to directly link to entities in web admin UI.
"

File: source/blog/2018-01-18-ovirt-admin-bookmarks.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### Examples
+
+* fqdn/ovirt-engine/webadmin#vms-general;NAME=MyVM will take you to general detail tab for the MyVM virtual machine.
```
Comment (vojtechszocs) [inline source/blog/2018-01-18-ovirt-admin-bookmarks.md]: For consistency, I'd suggest to drop the `fqdn/ovirt-engine/webadmin` URL prefix.

**Comment by vojtechszocs** [comment]
cc @jelkosz

**Comment by vojtechszocs** [comment]
Latest changes look good, +1 from me.

**Comment by gregsheremeta** [comment]
lgtm

**Pull request merged**